src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs

```diff
@@ -318,7 +318,7 @@ public static bool TryReadFrom(ref Utf8JsonReader reader, out JsonDocument docum
         ///   The current <paramref name="reader"/> token does not start or represent a value.
         /// </exception>
         /// <exception cref="JsonReaderException">
-        ///   A value could not be read from <paramref name="reader"/>.
```

Comment:
We use `paramref` above (in the `<para>` tag), why remove it from here? Or why not call out the type name itself (`Utf8JsonReader`)

Reply:
> We use paramref above (in the \<para\> tag), why remove it from here?

That was my attempt at addressing the feedback of "from the reader" (https://github.com/dotnet/corefx/pull/36350/files/7f82b7617e1b7ed072ac9c6e52776095ff7a395e#diff-2dd84f658be0fa2c0ed8c6af82c13260).

In my head, at least, if it's paramref it should just just say "from `reader`", since the paramref changes the semantic interpretation.  E.g.  "The file specified in `filename` was not found.", or if the name of the parameter was `readFrom`  then it would be "A value could not be read from `readFrom`.".